R-package/DESCRIPTION

```diff
@@ -5,7 +5,7 @@ Version: 2.1.0
 Date: 2018-01-25
 Author: Guolin Ke <guolin.ke@microsoft.com>
 Maintainer: Guolin Ke <guolin.ke@microsoft.com>
-Description: LightGBM is a gradient boosting framework that uses tree based learning algorithms. 
+Description: Tree based algorithms can be improved by introducing boosting frameworks. LightGBM is one such framework.
```

Comment:
This is in response to my least-favorite CRAN note. Your package's Description cannot start with the words "this package" or the name of the package.

Reply:
@jameslamb I think this Description is too short. We can copy some from https://github.com/Microsoft/LightGBM/blob/master/README.md

Reply:
@guolinke would you like me to make that change on this PR?

Reply:
yeah